javascript/features/gangs/gang_commands.js

```diff
@@ -458,6 +471,34 @@ class GangCommands {
                     message: formattedMessage
                 });
             });
+
+            menu.addItem('Member skin', gang.skinId, async() => {
```

Comment:
What does this display when `gang.skinId` is not set? `0`? Maybe we should have something like to make it look more consistent with other menus where there isn't a set value.

```javascript
menu.addItem('Member skin', gang.skinId ?? '-', async() => {
```

Reply:
As far as JS is aware it's null. So I assume it would result into 'null' what is indeed not intended.